django-rgd-imagery/tests/test_rest.py

```diff
@@ -10,49 +10,78 @@ def test_download_image_file(admin_api_client, astro_image):
     assert status.is_redirect(response.status_code)
 
 
+@pytest.mark.xfail
```

Comment:
I've marked two tests here as xfail. They shouldn't be failing and I cannot figure out what is different between these tests and what I do manually with swagger/the Python client.

I suspect something wacky is going on with the serializer where the m2m `source_images` field isn't saved when the serializer or view commits the record. @mcovalt, can you help me figure out what is different here?

Reply:
Deal with later